dds/idl/metaclass_generator.cpp

```diff
@@ -117,6 +117,13 @@ namespace {
     }
   }
 
+  std::string string_type(Classification cls)
+  {
+    return be_global->language_mapping() == BE_GlobalData::LANGMAP_CXX11 ?
+      ((cls & CL_WIDE) ? "std::wstring" : "std::string") :
```

Comment:
I'm not all that happy about the Codacy (cppcheck) changes.... but OK.  I wonder if we can override some of these?  Operator precedence rules exist for a reason and compilers are pretty good about warning if we get it wrong.

Reply:
Yeah, I thought this was overzealous as well. There's no way a compiler would mess up there.